Agenda/Modules/Summary/Views/SummaryTableViewCell.swift

```diff
@@ -76,7 +76,8 @@ final class SummaryTableViewCell: UITableViewCell {
     }
     
     public func configure(data: Summary) {
-        iconImageView.image = UIImage(systemName: data.iconImagePath, withConfiguration: UIImage.SymbolConfiguration(weight: .semibold))?.withTintColor(data.tintColor, renderingMode: .alwaysOriginal)
+//        iconImageView.image = UIImage(systemName: data.iconImagePath, withConfiguration: UIImage.SymbolConfiguration(weight: .semibold))?.withTintColor(data.tintColor, renderingMode: .alwaysOriginal)
```

Comment:
Remove comment from production / implement semibold configuration for new way of creating icons